cmake/Modules/FindN2P2.cmake

```diff
@@ -0,0 +1,47 @@
+include(FindPackageHandleStandardArgs)
+
+# Check if N2P2_DIR is set manually.
+if (DEFINED ENV{N2P2_DIR})
+  set(N2P2_DIR "${N2P2_DIR}")
+# If not, try if directory "lib/nnp/n2p2" exists.
+else()
+  get_filename_component(_fullpath "${LAMMPS_LIB_SOURCE_DIR}/nnp/n2p2" REALPATH)
```

Comment:
Not sure can rely on a manually downloaded package here, I would prefer a `ExternalProject_Add()`, have a look at `LATTE.cmake` as an example.